deploy/charts/openshift-routes/templates/rbac.yaml

```diff
@@ -41,11 +41,20 @@ rules:
 - apiGroups:
     - cert-manager.io
   resources:
-    - certificaterequests/status
+    - certificates/status
   verbs:
     - get
     - list
     - watch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
```

Comment:
We can probably remove the create permission.

Reply:
Oh yeah, cheers! Forgot this when I removed the `Owns(Secrets)`!